FORMAT.md

```diff
@@ -283,13 +300,60 @@ This is represented by a path between the same [long, lat] point on different le
 ```
 ---
 
-#### Disclaimer
+## Indoor Assets
+
+Indoor Assets are optional pieces of furniture you can add to an Indoor Map.  These are a separate GeoJSON format from the main Indoor Map format that defines all the Indoor Assets for a given floor, which can then be imported using the Import Assets function of the WRLD Indoor Map Designer.
+
+Each assets GeoJSON file is expected to contain a FeatureCollection in the WGS84 (EPSG: 4326) Coordinate Reference System, and include the appropriate crs attribute. Assets are represented as GeoJSON features with Polygon or Point geometry, with the center of the Polygon or the Point position itself being used as the central point of the Asset.
+
+### Asset Properties
+
+| Attribute| Type  | Description                                                                          |
+| -------- | ----- | ------------------------------------------------------------------------------------ |
+|`id`      |integer|An identifier for the feature. Each asset must have a unique id.
```

Comment:
Remove the id attribute as it's ignored by the import process (see https://eegeo-team.atlassian.net/browse/MPLY-11811)